src/main/java/com/hubspot/jinjava/doc/annotations/JinjavaDoc.java

```diff
@@ -11,6 +11,8 @@
 
   String value() default "";
 
+  JinjavaParam input();
```

Comment:
while you're in here, can you add a `required` attribute to `JinjavaParam`? @williamspiro wanted one.

Reply:
Labeled all filter/exp tests with this new boolean value.